app/models/identity.rb

```diff
@@ -1,21 +1,22 @@
 class Identity < OmniAuth::Identity::Models::ActiveRecord
+
+  DEFAULT_PASSWORD_EXPIRATION_PERIOD = \
+    Rails.configuration.accounts.default_password_expiration_period
+
   belongs_to :user, inverse_of: :identity
 
+  has_one :reset_code, dependent: :destroy, inverse_of: :identity
+
   attr_accessible :password, :password_confirmation
 
-  # We need these validations because omniauth-identity does not
-  # provide them by default...
-  validates :password, presence: true,
-                       length: {minimum: 8, maximum: 40}
+  # We need these validations because
+  # omniauth-identity does not provide them by default
+  # These attributes are (obviously) not saved in the database
+  validates :password, presence: true, length: { minimum: 8 }
```

Comment:
I can readd this if needed, but honestly there's no reason to limit password length I believe?